### PR TITLE
Update co.m3u

### PR DESCRIPTION
**Pull request by thetucan1**
- Update  Canal CNC Pasto
- Add Galeras Tv
- Add Hits Chanel
- Add sentimiento vallenato
- Add Son Corazon
- Add Son Latino
- Add Son Popular

----From https://www.cncpasto.com/
🟨🟦🟥▪▪▪▪▪

### CODE REVIEW

**Comment by what-a** [comment]
Not playing for me in California. Geo-blocked?

**Review by BellezaEmporium — APPROVED** [review]
Works for me, UK IP.

**Review by dct-infra — COMMENTED** [review]
all hvt links err on me, geoblocked? RU IP.

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]
on second testing, hvtrafico ddns links stopped working.

**Review by its0x08 — DISMISSED** [review]

File: streams/co.m3u [hunk 1]
```diff
@@ -133,5 +133,23 @@ https://hvtrafico.ddns.net/cinema720/cinema720.stream/playlist.m3u8
...
+#EXTINF:-1 tvg-id="",Hits Channel (Pasto | Nariño) (720p)
+https://hvtrafico.ddns.net/pastocnc1/cnc1/playlist.m3u8
+#EXTINF:-1 tvg-id="",Sentimiento Vallenato (Pasto | Nariño) (720p)
+https://hvtrafico.ddns.net/pastocnc4/cnc4/playlist.m3u8
+#EXTINF:-1 tvg-id="",Son Corazon (Pasto | Nariño) (720p)
+https://hvtrafico.ddns.net/pastocnc3/cnc3/playlist.m3u8
+#EXTINF:-1 tvg-id="",Son Latino (Pasto | Nariño) (720p)
+https://hvtrafico.ddns.net/pastocnc2/cnc2/playlist.m3u8
+#EXTINF:-1 tvg-id="",Son Popular (Pasto | Nariño) (720p)
+https://hvtrafico.ddns.net/pastocnc7/cnc7/playlist.m3u8
```
Comment (freearhey) [inline streams/co.m3u:155]: image

**Review by freearhey — CHANGES_REQUESTED** [review]

**Review by BellezaEmporium — DISMISSED** [review]

File: streams/co.m3u [hunk 2]
```diff
@@ -86,8 +84,8 @@ https://streamyes.alsolnet.com/tvgracia/live/playlist.m3u8
 https://stmv2.voxtvhd.com.br/tvnglobal/tvnglobal/playlist.m3u8
 #EXTINF:-1 tvg-id="UNOPLAY.co" status="online",UNO PLAY (720p) [Not 24/7]
 https://live-edge-bhs-1.cdn.enetres.net/184784E1D210401F8041E3E1266822CC021/playlist.m3u8
-#EXTINF:-1 tvg-id="" status="online",CTV barranquilla (Barranquilla | Atlántico) (720p)
-https://59a564764e2b6.streamlock.net/ctvbarranquilla/_definst_/ctv/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Canal 6 (Bogotà DC) (720p)
+http://190.93.137.61:1935/live/canal6/playlist.m3u8
```
Comment (freearhey) [inline streams/co.m3u:88]: image

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked link again.

**Review by BellezaEmporium — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**